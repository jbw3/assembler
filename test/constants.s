num = 2
hex=0xA1

label1:
    mov r1, r2
    mov r2, r3
label2:
    mov r3, r4

lbl = label2
    addi r0, num
    addi r1, hex
    addi r2, lbl
