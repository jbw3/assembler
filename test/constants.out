0112
0123
0134
c001
c051
c002
